_docs/request-matching.md

```diff
@@ -802,6 +802,37 @@ stubFor(
 
 JSON:
```

Comment:
Should I add a comment noting that this is only supported from version 3.4+?

Reply:
Done!

Reply:
thanks! Let's the pipeline succeed and I merge it